tests/rewards/test_reward_nets.py

```diff
@@ -75,13 +72,16 @@ def _make_env_and_save_reward_net(env_name, reward_type, tmpdir):
         net = reward_nets.NormalizedRewardNet(net, networks.RunningNorm)
     elif reward_type == "RewardNet_shaped":
         net = reward_nets.ShapedRewardNet(net, _potential, discount_factor=0.99)
+    assert (
```

Comment:
Does this function do the right thing for all reward types in the reward registry? If we add a reward to the registry, will we know to update this function?

Given this is just a helper for a *test*, I don't want to complicate things too much, but I might lean a bit towards actually just checking a hard-coded whitelist this helper was designed to work with.

Reply:
I've switch over to a hard coded white list.